package.json

```diff
@@ -133,5 +133,8 @@
     "react": "^16.4.0",
     "react-dom": "^16.4.0",
     "react-transition-group": "^2.5.0"
+  },
+  "resolutions": {
+    "js-yaml": "^3.13.1"
```

Comment:
@brandon-binary would you please specify which package is dependent on `js-yaml`?

Reply:
`babel-plugin-inline-react-svg`